src/CommandingOfficers/Bear_Bull.java

```diff
@@ -149,13 +159,18 @@ public String toString()
     protected void perform(MapMaster gameMap)
     {
       myCommander.addCOModifier(this);
+    }
 
+    @Override
+    public GameEventQueue getEvents(MapMaster gameMap)
+    {
       // Damage is dealt after swapping D2Ds so it's actually useful to Bear
-      COcast.liquidateMassDamage = true; // Collect money instead of ability energy
-      GameEvent damage = new MassDamageEvent(findVictims(gameMap), DOWNUPTURN_LIQUIDATION, false);
-      damage.performEvent(gameMap);
-      GameEventListener.publishEvent(damage);
-      COcast.liquidateMassDamage = false;
+      GameEventQueue powerEvents = new GameEventQueue();
+      powerEvents.add( (map) -> COcast.liquidateMassDamage = true ); // Collect money instead of ability energy
```

Comment:
This seems... silly. Using a reified lambda to hand a function manipulating internal state to another entity just breaks... everything about OO best practices.
This also isn't a `GameEvent` in any real sense - it could live quite happily (and more sensibly) in `perform()`.

Reply:
OK yeah, you can throw `COcast.liquidateMassDamage = true;` in `perform()`, and then toss `COcast.liquidateMassDamage = false;` in `endTurn()` to prevent gaining funds from mass damage events during other CO's turns.
I guess that leaves open the possibility of _another_ mass damage event during your turn that triggers more income accrual, but I mean, that's kind of a non-thing.

Reply:
It also spreads around the code more broadly, and ties the sequencing of it to other systems.
Putting all three actions next to each other in the event queue makes a lot of semantic (minimizing side-effects) and idiomatic (putting things that happen together and are related together) sense.

Reply:
Yeah, fair. Actually, there's no reason you can't just calculate the income up front and inject a `CanHazMoneyPlz` (or maybe just like, `ModifyFundsEvent`, even) event instead of waiting for the damage events to be published. That seems cleaner than either previous idea.

Reply:
Uh. Hm. That's pretty reasonable, and theoretically simpler than what I had before.